dm/ui/src/pages/migration/task/index.tsx

```diff
@@ -459,6 +511,12 @@ const TaskList: React.FC = () => {
     refetchCurrentTaskStatus()
   }, [data])
 
+  useEffect(() => {
+    if (!isCreateTaskModalVisible) {
+      form.resetFields()
+    }
+  }, [isStartTaskModalVisible])
```

Comment:
Is it a typo? the hook dependency is `isStartTaskModalVisible` but body is `isCreateTaskModalVisible`.

Reply:
yeah, typo